cumulus/parachains/runtimes/coretime/coretime-rococo/src/coretime.rs

```diff
@@ -135,6 +135,7 @@ impl CoretimeInterface for CoretimeAllocator {
 			Instruction::Transact {
 				origin_kind: OriginKind::Native,
 				call: request_core_count_call.encode().into(),
+				fallback_max_weight: Some(Weight::from_parts(1_000_000_000, 200_000)),
```

Comment:
This seems reasonable.
The calls should have a fallback in this case, as the order of upgrade is not guaranteed and it's a bidirectional interface. On Westend it would be possible to do relay -> coretime to enable us to have no fallback on the coretime chain and only keep fallbacks for the relay.
But I think it's better to do this the way we'll do it on Kusama/Polkadot so let's do fallbacks to previous values.